Engine/Fuego/Events/ApplicationEvent.h

```diff
@@ -13,11 +13,11 @@ class FUEGO_API WindowResizeEvent : public Event
     {
     }
 
-    inline unsigned int GetWidth() const
+    unsigned int GetWidth() const
```

Comment:
In general I would recommend you to use fix sized type aliases like `uint32_t`

Reply:
Fixed